src/AddTodoForm.js

```diff
@@ -1,11 +1,30 @@
-import React from "react";
+import React, {useState} from "react";
+
+
+let AddTodoForm = ({onAddTodo}) => {
+
+  const[inputText, setInputText] = useState("");
+  
+
+  let handleAddTodo = function (event){
+    event.preventDefault(); // Prevent the default behaviour of the form submit.
+
+    // Retrieves the value of the title element from 
+    // the evenet target and store it in a variable named todoTitle.
+    let todoTitle = inputText; 
+    console.log(todoTitle);
+    setInputText(event.target.value);
+    setInputText(""); // Reset form onSubmit.
+    onAddTodo(todoTitle); // Invokes onAddTodo callback prop and pass the todoTitle as an argument inside this handleAddTodo function
+    
+  };
+
 
-const AddTodoForm = () => {
   return (
-    <form>
+    <form onSubmit={(event) => handleAddTodo(event)}>
         <label htmlFor="todoTitle">Title</label>
-        <input id="todoTitle"></input>
-        <button>Add</button>
+        <input id="todoTitle" name="title" value = {inputText} onChange={(event) => setInputText(event.target.value)}></input>
```

Comment:
While this is a valid way to get data out of an input, the instructions want us to only use an event handler on the form element. Feel free to remove the value and onChange prop on the input element

Reply:
I will do that.

Thanks.